### PR TITLE
Fix bridge app path in tests yaml (bridge-app does not exist, only bridge)

### PR DESCRIPTION
**Pull request by andy31415**
The target `bridge-app` does not exist, only `bridge`. Also oddly enough the tests attempted to run bridge app tests against something in TV output, which seems wrong.

I currently do not believe we use bridge app at all ... unsure what the path forward is here, but at least this fixes paths.

### CODE REVIEW

File: .github/workflows/darwin-tests.yaml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
                         --target darwin-x64-ota-provider-${BUILD_VARIANT} \
                         --target darwin-x64-ota-requestor-${BUILD_VARIANT} \
                         --target darwin-x64-tv-app-${BUILD_VARIANT} \
-                        --target darwin-x64-bridge-app-${BUILD_VARIANT} \
+                        --target darwin-x64-bridge-${BUILD_VARIANT} \
```
Comment (bzbarsky-apple) [inline .github/workflows/darwin-tests.yaml:97]: Hrm.  I'm surprised this passed if the target does not exist.  :(

File: .github/workflows/darwin-tests.yaml [hunk 2]
```diff
@@ -113,7 +113,7 @@ jobs:
                      --ota-provider-app ./out/darwin-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
                      --ota-requestor-app ./out/darwin-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
                      --tv-app ./out/darwin-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/darwin-x64-tv-app-${BUILD_VARIANT}/chip-bridge-app \
+                     --bridge-app ./out/darwin-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
```
Comment (bzbarsky-apple) [inline .github/workflows/darwin-tests.yaml:116]: Yikes, copy/paste errors.  I thought I had caught all those.

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
> I currently do not believe we use bridge app at all ... unsure what the path forward is here, but at least this fixes paths.

The path forward is to change some of the relevant YAMLs from manual to automated.

File: .github/workflows/darwin-tests.yaml (hunk 1 above)
Comment (andy31415) [reply .github/workflows/darwin-tests.yaml:97]: This confused me too .... either TV does build a bridge app (doubt it) or we just do not run bridge tests.

Comment (bzbarsky-apple) [reply .github/workflows/darwin-tests.yaml:97]: Right now we do not run bridge tests.

Comment (andy31415) [reply .github/workflows/darwin-tests.yaml:97]: I think currently we just log a 'unknown variant'. Fixing that is probably a separate concern.

**Pull request merged**